src/Oscar.jl

```diff
@@ -274,6 +291,17 @@ function build()
 end
 
 
+@doc Markdown.doc"""
+    test_module(x, new::Bool = true)
+
+Run the Oscar tests in the file `x.jl` where `x` is a string.
+
+If `x == "all"` run the entire testsuite.
+
+The optional parameter `new` takes the values `false` and `true` (default). If
+`true`, then the tests are run in a new session, otherwise the currently active
+session is used.
+"""
 function test_module(x, new::Bool = true)
```

Comment:
Should we specify the type of `x` ?
```suggestion
function test_module(x::AbstractString, new::Bool = true)
```